v2/tools/generator/internal/codegen/pipeline/report_resource_versions.go

```diff
@@ -92,26 +93,25 @@ func (report *ResourceVersionsReport) WriteTo(outputPath string, samplesURL stri
 	report.WriteToBuffer(&buffer, samplesURL)
 
 	if _, err := os.Stat(outputPath); os.IsNotExist(err) {
-		err = os.MkdirAll(outputPath, 0700)
+		err = os.MkdirAll(outputPath, 0o700)
 		if err != nil {
 			return errors.Wrapf(err, "Unable to create directory %q", outputPath)
 		}
 	}
 
 	destination := path.Join(outputPath, "resources.md")
-	return ioutil.WriteFile(destination, []byte(buffer.String()), 0600)
+	return ioutil.WriteFile(destination, []byte(buffer.String()), 0o600)
```

Comment:
Is the leading `0` not sufficient to specify Octal?

Reply:
auto-formatter did this (to make it clearer, leading `0` is not obvious!)